mailpoet/lib/Automation/Engine/Data/WorkflowRunLog.php

```diff
@@ -0,0 +1,175 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Engine\Data;
+
+use DateTimeImmutable;
+use InvalidArgumentException;
+use MailPoet\Automation\Engine\Exceptions\InvalidStateException;
+use MailPoet\Automation\Engine\Exceptions\UnexpectedValueException;
+use MailPoet\Automation\Engine\Utils\Json;
+use Throwable;
+
+class WorkflowRunLog {
+
+  const STATUS_RUNNING = 'running';
```

Comment:
We might be able to just delete this status. I don't think it's meaningful, but if there's a use case for having it please let me know.

Reply:
I guess if we don't have a use for it yet, we can indeed delete it.